taxonomies/ingredients_processing.txt

```diff
@@ -894,3 +934,5 @@ es:sin hueso
 
 en:inflated
 es:hinchado,hinchada,hinchados,hinchadas
+
+#en:for color, added for color, colored with
```

Comment:
Those are in fact additives classes, we will deal with it in a differenty way, but it needs some changes to the parsing code.

Reply:
That's why I put those in comments. I am just gathering all adjectives.